### PR TITLE
Update User Guide and Developers Guide

### PR DESCRIPTION
**Pull request by jhengy**
1. removed Original Ui.png
2. updated User Guide: edited Section 2 and added 7 features to Section 3
3. updated Developer Guide: added 4 use cases to Appendix D

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -46,113 +46,107 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
 === Viewing help : `help`
 
 Format: `help`
 
-=== Adding a person: `add`
+=== Adding an entry: `addEntry`
 
 Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Format: `AddEntry ~SECTION [#TAG]... t/TITLE [s/SUBHEADER] d/DURATION`
```
Comment (anubh-v) [inline docs/UserGuide.adoc]: Let's standardise: either addEntry or AddEntry

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -46,113 +46,107 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `addEntry ~experience #java t/The Source Enterprise s/Java Programmer intern d/ May 2010 - Aug 2010`
+* `addEntry ~awards #java`
 
-=== Listing all persons : `list`
+=== Adding a bullet point: `addDescription`
 
-Shows a list of all persons in the address book. +
-Format: `list`
+Adds a bullet point under the description of an entry in the resuMaker +
```
Comment (anubh-v) [inline docs/UserGuide.adoc]: Let's standardise as "ResuMaker" - not resuMaker

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -46,113 +46,107 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
+Format: `addBullet CONTENTTOADD`
 
-=== Editing a person : `edit`
+Examples:
 
-Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+* `addBullet  research design and implement scalable java`
```
Comment (anubh-v) [inline docs/UserGuide.adoc]: Is the command `addBullet` or `addDescription` ? 
Also what does `research design and implement scalable java` mean?

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -46,113 +46,107 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
-Selects the 1st person in the results of the `find` command.
+`delete 2` +
+Deletes the 2nd entry in the address book.
+* `list ~education` +
+`delete 1` +
+Deletes the 1st entry displayed under education section
+
 
 
 === Creating a SoC / NUS Event (Project / Work Experience / Skill): `nus`
```
Comment (anubh-v) [inline docs/UserGuide.adoc]: Seems like the rest of the features aren't rendered correctly. Is it my mistake in the asciidoc for my part?

**Comment by anubh-v** [comment]
You can view whitespace characters in notepad++, in order to fix the whitespace style errors.
See:

File: docs/UserGuide.adoc (hunk 3 above)
Comment (jhengy) [reply docs/UserGuide.adoc]: addBullet is a bullet point under description. Description is a list of bullet points

Comment (jhengy) [reply docs/UserGuide.adoc]: Not so sure of the meaning,  but I suppose it does not matter? Since it's just an example

**Pull request merged**